### DISCUSSION TITLE
Flex and Groups are not working properly.

### DISCUSSION THREAD

**Discussion by wdmswapnilpatil** [mantinedev/mantine#5205] [Q&A]
When I add flex or groups in 7.2.0 then display:flex property not getting applied. I checked it on documentation but there some external css file (13e1088eca5aa9f4.css) is adding display: flex. I added postcss postcss mantine also imported '@mantine/core/styles.css';

**Comment by rtivital** [comment]
It works correctly on codesandbox – https://codesandbox.io/s/epic-solomon-67rrjz?file=/src/App.tsx
Make sure that you are using one of the supported frameworks or load styles file properly if you use anything else.

**Reply by wdmswapnilpatil** [reply]
How to load styles file if I am using anything else?

**Reply by rtivital** [reply]
Follow documentation for your framework/build tool – https://mantine.dev/getting-started/
If your framework/build tool is not on the docs website, find documentation for it on the website of the tool

**Comment by alecmce-maltedai** [comment]
I had the same issue. For anyone looking, I think this means that `import '@mantine/core/styles.css';` was omitted.
